src/fetchers/MainPage/MainPage.tsx

```diff
@@ -16,20 +20,38 @@ import DataCards from '../../Articles.json';
 import Graphs from '../../Graphs.json';
 import TabletFilter from '../../components/Tablet&Mobile/FilterComponent/Filter';
 import { HeadLinesFilters, EverythingFilters,FILTER_OPTIONS } from '../../FiltersData';
-
-
+import FilterTablet from '../../components/Tablet&Mobile/FilterTablet/FilterTablet';
+import {ButtonProps} from '../../components/Button/Button';
 const data =["#Covid-19", "Israel", "+2"];
 
 const navbarProps = {
   recentSearches: ["crypto", "soccer", "soccer"],
 }
 
+const subFilterList = [
```

Comment:
You'r main page component is a bit messy, you need to take all of the code thats related to the filter and put it in another component.
It will make the code much cleaner and will help organize it